tox.ini

```diff
@@ -1,12 +1,20 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py27,py34,py35,py36,py37,pypy,flake8
+envlist = py35
+skipsdist = true
+
 
 [testenv]
-deps = pytest>=3.0
-commands = pytest {posargs:tests}
+deps =
+    -r{toxinidir}/requirements/test.txt
+commands =
+    pip install -e .
```

Comment:
Wouldn't you get about the same effect if you removed this and didn't use `skipsdist`?

Reply:
did not know that, thank you for the suggestion